book/modules/appendix2.tex

```diff
@@ -513,9 +670,41 @@
 	With our newfound geometric intuition, we can understand solutions to systems of linear equations in a different way.
 	The solution set to a system of linear equations of two variables must be the result of intersecting lines. Therefore,
 	the only options are: a point, a line, or the empty set. The solution to a system of linear equations of three variables
-	is similarly restricted. I can only be: a point, a line, a plane, or the empty set.
-
-	XXX Figure of three planes intersecting in a line and three planes pairwise-intersecting, but not mutually intersecting.
+	is similarly restricted. It can only be: a point, a line, a plane, or the empty set.
+	
+	\begin{center}
+		\begin{tikzpicture}
+			\begin{axis}[
+				view={10}{20},
+				z buffer=sort,
+				hide axis,
+			]
+			\addplot3 [surf,domain=-5:0,y domain=-10:10,samples=2, opacity=.8, mygreen] ({x}, {y}, {x/2});
+			\addplot3 [surf,domain=-5:0,y domain=-10:10,samples=2, opacity=.8, myorange] ({x}, {y}, {0});
+			\addplot3 [surf,domain=-5:0,y domain=-10:10,samples=2, opacity=.9, WildStrawberry] ({x}, {y}, {-x/2});
+			\addplot3 [surf,domain=0:5,y domain=-10:10,samples=2, opacity=.9, WildStrawberry] ({x}, {y}, {-x/2});
+			\addplot3 [surf,domain=0:5,y domain=-10:10,samples=2, opacity=.8, myorange] ({x}, {y}, {0});
+			\addplot3 [surf,domain=0:5,y domain=-10:10,samples=2, opacity=.8, mygreen] ({x}, {y}, {x/2});
+			\end{axis}
+		\end{tikzpicture}
+		~~~~~
```

Comment:
prefer `\hspace{.5in}` (or whatever the appropriate length is) instead of a bunch of ~~~